test/run_cli.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const ChildProcess = require('child_process');
+const Path = require('path');
+
+const labPath = Path.join(__dirname, '..', 'bin', '_lab');
+
+module.exports = (args, callback) => {
+
+    return new Promise((resolve, reject) => {
+
+        const cli = ChildProcess.spawn('node', [labPath, ...args]);
```

Comment:
It’s the spread operator, but you are right: I just saw that it’s [only supported in node 5, not in 4](https://kangax.github.io/compat-table/es6/).